Runtime/Input/InputController.cs

```diff
@@ -9,14 +10,25 @@ namespace Innoactive.Creator.Core.Input
     /// <summary>
     /// Central controller for input via the new Input System using C# events.
     /// </summary>
+    [RequireComponent(typeof(PlayerInput))]
     public class InputController : UnitySceneSingleton<InputController>
     {
+        public class InputEventArgs : EventArgs
+        {
+            public readonly object Context;
+
+            public InputEventArgs(object context)
+            {
+                Context = context;
+            }
+        }
+
         private struct ListenerInfo
         {
             public readonly IInputActionListener ActionListener;
-            public readonly Action<InputAction.CallbackContext> Action;
+            public readonly Action<InputEventArgs> Action;
```

Comment:
This is an API break, tbh there shouldn't be any real issue, but let's remember it.